inference-engine/src/preprocessing/cpu_x86_avx512/ie_preprocess_gapi_kernels_avx512.hpp

```diff
@@ -26,34 +26,34 @@ void calcRowArea_32F(float dst[], const float *src[], const Size &inSz, const Si
                      const float xalpha[], float vbuf[]);
 
 #if USE_CVKL
-void calcRowArea_CVKL_U8_SSE42(const uchar  * src[],
-                                     uchar    dst[],
-                               const Size   & inSz,
-                               const Size   & outSz,
-                                     int      y,
-                               const uint16_t xsi[],
-                               const uint16_t ysi[],
-                               const uint16_t xalpha[],
-                               const uint16_t yalpha[],
-                                     int      x_max_count,
-                                     int      y_max_count,
-                                     uint16_t vert_sum[]);
+void calcRowArea_CVKL_U8(const uchar  * src[],
+                                 uchar    dst[],
```

Comment:
Seems like indentation is broken a little

Reply:
done